core/domain/services/graphql/connection_resolvers/AttendeeConnectionResolver.php

```diff
@@ -99,14 +100,43 @@ public function get_query_args()
         if (! empty($this->args['where'])) {
             $input_fields = $this->sanitizeInputFields($this->args['where']);
 
+            // Since we do not have any falsy values in query params
+            // Lets get rid of empty values
+            $input_fields = array_filter($input_fields);
+
             // Use the proper operator.
+            if (! empty($input_fields['Registration.Event.EVT_ID']) && is_array($input_fields['Registration.Event.EVT_ID'])) {
+                $input_fields['Registration.Event.EVT_ID'] = ['IN', $input_fields['Registration.Event.EVT_ID']];
+            }
             if (! empty($input_fields['Registration.Ticket.TKT_ID']) && is_array($input_fields['Registration.Ticket.TKT_ID'])) {
-                $input_fields['Registration.Ticket.TKT_ID'] = ['in', $input_fields['Registration.Ticket.TKT_ID']];
+                $input_fields['Registration.Ticket.TKT_ID'] = ['IN', $input_fields['Registration.Ticket.TKT_ID']];
             }
-            // Use the proper operator.
-            if (! empty($input_fields['Registration.Event.EVT_ID']) && is_array($input_fields['Registration.Event.EVT_ID'])) {
-                $input_fields['Registration.Event.EVT_ID'] = ['in', $input_fields['Registration.Event.EVT_ID']];
+            // If Ticket param is passed, it will have preference over Datetime param
+            // So, use Datetime param only if a Ticket param is not passed
+            if (! empty($input_fields['Datetime.DTT_ID']) && empty($input_fields['Registration.Ticket.TKT_ID'])) {
+                $datetimeIds = $input_fields['Datetime.DTT_ID'];
+                // Make sure it's an array, ready for "IN" operator
+                $datetimeIds = is_array($datetimeIds) ? $datetimeIds : [$datetimeIds];
+
+                try {
+                    // Get related ticket IDs for the given dates
+                    $ticketIds = EEM_Ticket::instance()->get_col([
+                        [
+                            'Datetime.DTT_ID' => ['IN', $datetimeIds],
+                            'TKT_deleted'     => ['IN', [true, false]],
+                        ],
+                        'default_where_conditions' => 'minimum',
+                    ]);
+                } catch (\Throwable $th) {
```

Comment:
plz import all classes, ie: add use statement at top of class for this, then simplify FQCN here

Reply:
Done